tests/common/scripts/cleaning/test_clean_epoch_dateTime.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from common.scripts.cleaning import clean_epoch_dateTime
+
+
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        ("40246871", "1971-04-11T20:41:11+02:00"),
+        ("1330063200000", "2012-02-24T07:00:00+02:00"),
+        ("1330695360000", "2012-03-02T14:36:00+02:00"),
```

Comment:
the case where the input is `""` or where `datetime.datetime.fromtimestamp(float(raw_input))` raises a `ValueError` doesn't seem to be tested ?

Reply:
These test have been added!
Do you agree that we should return "" if there is a ValueError"?

Reply:
MSSQL failed again..